### PR TITLE
Docs-3943 TC 2.2 Update SAML Article and Fix Image Links

### PR DESCRIPTION
**Pull request by micjohnson777**
This commit splits the current SAML.md article into to separate articles SAMLAD.md for the Active Directory procedure and SAMLGA for the Google Admin procedure.
It fixes issue with image links that were not displaying, and fixes the flow and headers to present the information better and make the content more readable.

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/TrueCommand/Administration/Settings/SAMLAD.md [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+
+1. Log in to your TrueCommand system (i.e., server, container, VM). 
+   To access the TrueCommand web interface, enter http://*IP:PORT* where *IP:PORT* is the IP address and port number assigned to your TrueCommand system, into a web browser URL field and then press <kbd>Enter</kbd>. 
+
+2. Go to **Settings > Administration**, then click on **Configure** in the **Configuration** widget. 
+   The **Configuration** screen with editable settings displays.
+
+   {{< hint ok >}}
+   The **SSL Options** on the **Configuration** settings form has **Accept self-signed certificates** selected by default. 
+   Keep this setting to be able to use a self-signed certificate. It also helps prevent issues with http verses https URLs.
```
Comment (tonyriv3) [inline content/TrueCommand/Administration/Settings/SAMLAD.md:30]:
```suggestion
   Keep this setting to be able to use a self-signed certificate. It also helps prevent issues with HTTP versus HTTPS URLs.
```

File: content/TrueCommand/Administration/Settings/SAMLAD.md [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+      | Telephone-Number | telephoneNumber |
+      | Title | title |
+      
+    i. Click **Finish**, then **Apply**, and finally **OK**.
+
+8. Close **Active Directory**.
+
+## Logging into TrueCommand SAML
+
+Go to the TrueCommand login page and click on **SAML Login** option to log in.
```
Comment (tonyriv3) [inline content/TrueCommand/Administration/Settings/SAMLAD.md:214]:
```suggestion
Go to the TrueCommand login page and click the **SAML Login** option to log in.
```

File: content/TrueCommand/Administration/Settings/SAMLGA.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+## Overview of the Google Admin and TrueCommand SAML Service Configuration Process
+
+To configure Google Admin as the IdP, you must:
+
+* Create a new App for SAML 
+* Configure the SAML app properties to act as the IdP service. 
+* Add the TrueCommand IP and port number as the ACS URL.
+* Configure the SAML app LDAP attributes properties
+
+After you configure SAML in Google Admin you then configure and start TrueCommand SAML service.
```
Comment (tonyriv3) [inline content/TrueCommand/Administration/Settings/SAMLGA.md:29]:
```suggestion
After you configure SAML in Google Admin, you configure and start the TrueCommand SAML service.
```

**Review by tonyriv3 — CHANGES_REQUESTED** [review]

File: content/TrueCommand/Administration/Settings/SAMLAD.md [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+   The **Configuration** screen with editable settings displays.
+
+   {{< hint ok >}}
+   The **SSL Options** on the **Configuration** settings form has **Accept self-signed certificates** selected by default. 
+   Keep this setting to be able to use a self-signed certificate. It also helps prevent issues with HTTP versus HTTPS URLs.
+   {{< /hint >}}
+
+3. Enter the URL from Active Directory.
+   Enter http://*ds.yourcompany.net*/FederationMetadata/2007-06/FederationMetadata.xml in the **SAML Identity Provider URL** field, then click **Save**. 
+
```
Comment (BonnieF) [inline content/TrueCommand/Administration/Settings/SAMLAD.md:35]: Click on **Configure** in the **Configuration** widget.

File: content/TrueCommand/Administration/Settings/SAMLAD.md [hunk 5]
```diff
@@ -0,0 +1,216 @@
...
+   b. Edit the certificate. 
+
+      1. Open a File Explorer window and locate the <file>tc.cer</file> file in <file>C:/local data/user/QE</file>.
+
+         ![LocateTcCertificateWithFileExplorer](/images/SAML/LocateTcCertificateWithFileExplorer.png "Locate TC Certificate With File Explorer")
+      
+      2. Select the <file>tc.cer</file> file, right-click, then select **Open with Notepad**.
+
+      3. Delete everything before the certificate open tag `<x509Certificate>`.
```
Comment (BonnieF) [inline content/TrueCommand/Administration/Settings/SAMLAD.md:101]: My instructions, in the test case say:
Delete everything except for what is inside X509Certificate, including these tags.

File: content/TrueCommand/Administration/Settings/SAMLAD.md [hunk 6]
```diff
@@ -0,0 +1,216 @@
...
+
+      ![SelectRelyingPartyTrustAndProperties](/images/SAML/SelectRelyingPartyTrustAndProperties.png "Select Relying Party Trust And Properties")
+   
+   b. Select the **Endpoints** tab, then click **Edit** to open the **Edit Endpoint** window.
+
+   c. Change the **Index** value to **1** and click **OK**. 
+      
+      ![EditPropertiesEndpointsChangeIndexValueCropped](/images/SAML/EditPropertiesEndpointsChangeIndexValueCropped.png "Edit Properties Endpoints Change Index Value Cropped")
+
+   d. Click the **Add SAML** button to open the **Add an Endpoint** window.
```
Comment (BonnieF) [inline content/TrueCommand/Administration/Settings/SAMLAD.md:163]: My test directions:
Click **Add SAML**
Select Endpoint type **SAML Logout**.

File: content/TrueCommand/Administration/Settings/SAMLAD.md [hunk 7]
```diff
@@ -0,0 +1,216 @@
...
+      | Telephone-Number | telephoneNumber |
+      | Title | title |
+      
+    i. Click **Finish**, then **Apply**, and finally **OK**.
+
+8. Close **Active Directory**.
+
+## Logging into TrueCommand SAML
+
+Go to the TrueCommand login page and click on **SAML Login** option to log in.
```
Comment (BonnieF) [inline content/TrueCommand/Administration/Settings/SAMLAD.md:214]: Go to the TrueCommand login page and **SAML Login** to log in.

**Review by BonnieF — DISMISSED** [review]
I left some comments with possible changes were the steps on the doc didn't match the steps in the test case.

File: content/TrueCommand/Administration/Settings/SAMLAD.md (hunk 6 above)
Comment (BonnieF) [reply content/TrueCommand/Administration/Settings/SAMLAD.md:163]: If this is changes you may need to change the screenshot for the 2nd endpoint "SAML Logout".

File: content/TrueCommand/Administration/Settings/SAMLAD.md (hunk 5 above)
Comment (micjohnson777) [reply content/TrueCommand/Administration/Settings/SAMLAD.md:101]: Yes, but that URL was just inside the open certificate tag so I had to find a way to describe this and cover that URL if it happens for the user.

File: content/TrueCommand/Administration/Settings/SAMLAD.md (hunk 6 above)
Comment (micjohnson777) [reply content/TrueCommand/Administration/Settings/SAMLAD.md:163]: I went back to the original instructions from JT's test script and the process we worked out and selecting Endpoint type **SAML Logout** that was not part of the instructions so this is new and probably why I missed it. I reviewed the original images in sequence and for this part there is no place to select the type **SAML Logout**

File: content/TrueCommand/Administration/Settings/SAMLAD.md (hunk 4 above)
Comment (micjohnson777) [reply content/TrueCommand/Administration/Settings/SAMLAD.md:35]: Save dumps you out so you have to click Configure again to get back into the Configuration screen.

File: content/TrueCommand/Administration/Settings/SAMLAD.md [hunk 8]
```diff
@@ -0,0 +1,218 @@
...
+      ![WizardSelectEnableSupportForSAML2_0WebSSOProtocolCropped](/images/SAML/WizardSelectEnableSupportForSAML2_0WebSSOProtocolCropped.png "Wizard Select Enable Support For SAML 2.0 Web SSO Protocol")
+
+   b. Enter the TrueCommand login URL (http://*IP:PORT*/saml/acs) in the **Relying party SAML 2.0 SSO service URL** field. 
+      *IP:PORT* is your TrueCommand system IP and port address.
+
+   c. Click **Next** to continue to the **Configure Identifiers** window.
+
+5. Configure the SAML identifiers. 
+   
+   a. Type **truecommand-sml** into the **Relying party trust identifier** field and click **Add**.
```
Comment (BonnieF) [inline content/TrueCommand/Administration/Settings/SAMLAD.md:140]: Type **truecommand-sml** should be
 Type **truecommand-saml**

Comment (micjohnson777) [reply content/TrueCommand/Administration/Settings/SAMLAD.md]: fixed I thought.

File: content/TrueCommand/Administration/Settings/SAMLAD.md [hunk 9]
```diff
@@ -0,0 +1,218 @@
...
+      ![WizardTypeDisplayNameCropped](/images/SAML/WizardTypeDisplayNameCropped.png "Wizard Type Display Name")
+   
+   e. Click **Next** to move on to the **Configure Certificate** window.
+
+3. Modify the TrueCommand Certificate (<file>tc.cer</file>). 
+
+   a. Open PowerShell and type the command `invoke-webRequest -uri http://IP:PORT/saml/metadata -outfile tc.cer`. 
+      *IP:PORT* is your TrueCommand system IP address/port number.
+
+      ![TC2.2PowershellInvokeCommand](/images/SAML/TC2.2PowershellInvokeCommand.png "Open Powershell Type Invoke Command")
```
Comment (BonnieF) [inline content/TrueCommand/Administration/Settings/SAMLAD.md:94]: TC2.2PowershellInvokeCommand
i showing as a broken link

**Review by BonnieF — CHANGES_REQUESTED** [review]
I am still showing that image as a broken link. I tried it in an "incognito" window as well.

**Review by BonnieF — APPROVED** [review]

**Pull request merged**